docs/flatfield/source/main.rst

```diff
@@ -17,17 +17,17 @@ flat-field consists of a single image model, the region corresponding to each
 slit in the science data is extracted on-the-fly from the flat-field data and
 applied to the corresponding slit in the science data.
 
-Multiple-integration datasets (the _rateints.fits products from the ramp_fit 
+Multiple-integration datasets (the _rate.fits products from the ramp_fit
```

Comment:
Actually the input product type for a multiple-integration dataset will in fact be the _rateints.fits file from ramp_fit, so that part of the old statement was correct.